package.json

```diff
@@ -1,116 +1,1963 @@
+    "VV Coin",
+    "Velix ID Token",
+    "Vixen Token",
+    "Vexareum Cash",
+    "VAYLA Token",
+    "Wrapped 0xEthereum Token",
+    "W12 Token",
+    "WatertoWorld",
+    "W3C",
+    "WABnetwork",
+    "WAR Token",
+    "Waves",
+    "WhalesburgToken",
+    "WincashCoin",
+    "WEED TOKEN",
+    "Webcoin",
+    "WEBN",
+    "Welcome Coin",
+    "WELL Token",
+    "Wrapped Ether",
+    "WFee",
+    "wegen",
+    "WorldGamingTech",
+    "ProjectWITH",
+    "WCOIN",
+    "WINSTEX",
+    "WinCoin",
+    "WIT",
+    "WITH coin",
+    "WANKCOIN",
+    "Win Last Mile",
+    "WeMediaCash",
+    "WemarkToken",
+    "Woonk",
+    "WiseWolf",
+    "woolchain",
+    "Woonkly",
+    "WOTChain",
+    "WePower Token",
+    "Worldcore",
+    "WisdomStock",
+    "ETHWSS Coin",
+    "WeToken",
+    "Walton Token",
+    "WITEE",
+    "Welltrado token",
+    "Web token Pay",
+    "WALLETOURS",
+    "Official USD token",
+    "Wizzle Infinity Token",
+    "Silver (Troy Ounce)",
+    "Appics",
+    "Gold (Troy Ounce)",
+    "Xaurum",
+    "XBANC",
+    "BlitzPredict",
+    "BIT DINERO",
+    "XCELTOKEN",
+    "ClearCoin",
+    "Connect coin(XCON)",
+    "Counterparty",
+    "XChain Token",
+    "XCOYNZ Token",
+    "XDAC COIN",
+    "XDMC token",
+    "NEM",
+    "XENIACOIN",
+    "Fructus",
+    "XI Network",
+    "Mixin",
+    "XCELTOKEN PLUS",
+    "Ethereum Lite Cash",
+    "STELLAR CLASSIC",
+    "STELLARGOLD",
+    "STELLAR CLASSIC",
+    "MingoCoin",
+    "XMED Chain Token",
+    "Monero Diamond",
+    "MOO token",
     "Monero",
-    "XMR",
-    "PrimeCoin",
-    "XPM",
-    "Spectrecoin",
-    "XSPEC",
+    "Monero Gold",
+    "XMAX",
+    "Ink Protocol",
+    "XENON",
+    "West African CFA Franc",
+    "PolarisToken",
+    "Proton",
+    "Payshares",
+    "Paytokens",
+    "Polaris Universal",
+    "MasterXriba",
+    "Rainbow",
+    "Aerum",
+    "XRP CLASSIC",
+    "Xchangerate Coin",
+    "XRT Token",
+    "XRT Token",
+    "XSAPPHIRE",
+    "Stakenet",
+    "SpaikCoin",
+    "SucreCoin",
+    "XstarToken",
+    "XTerato",
+    "XTRLPay",
+    "XtockToken",
     "VERGE",
-    "XVG",
+    "Vantage Token",
+    "XY Oracle",
+    "ZCoin",
+    "JamaiCoin",
+    "Yanu",
+    "Yaapoo",
+    "YouDeal Token",
+    "Yee - A Blockchain-powered & Cloud-based Social Ecosystem",
+    "On-Chain Verification",
+    "YesManCoin",
+    "Yooba token",
+    "ySignToken",
+    "Yellow Tether",
+    "YoozCoin",
+    "ZAG Token",
+    "South African Rand",
+    "ZBToken",
+    "ZBANK",
+    "ZBANK",
+    "0chain",
+    "Zloadr Token",
+    "Zloadr Token",
     "ZCash",
-    "ZEC",
+    "ZEEW",
+    "Horizen",
+    "ZEROCoin",
+    "ZEWT",
+    "Zuflo Coin",
+    "Ziggurat Token",
+    "Zilliqa",
+    "Zing Token",
+    "Zipper",
+    "ZJLT Distributed Factoring Network",
+    "Zilla Token",
+    "Zenswap Network Token",
+    "ZOM",
+    "ZperToken",
+    "0x Protocol Token",
+    "Zeus Shield Coin",
+    "Zulu Republic Token",
+    "0xcert Protocol Token",
```

Comment:
that's a long list

Reply:
Yeah, not sure yet about this, it is a little distasteful, but there is no official limit... on the other hand, people who are looking for these projects (which are niche) this is a good thing.